src/components/UserDashboard/MySavedQueries.js

```diff
@@ -144,92 +128,36 @@ const MySavedQueries = compose(
             .slice()
             .sort((a, b) => b.date - a.date)
             .map(q => (
-              <div
-                key={q.id}
-                css={`
-                  display: flex;
-                  padding: 10px 10px 10px 25px;
-                  border: 1px solid #e0e1e6;
-                  border-bottom: 0;
-                  transition-property: opacity;
-
-                  ${deletingIds.includes(q.id) &&
-                    `opacity: 0.6;
-            pointer-events: none;`} &:last-child {
-                    border-bottom: 1px solid #e0e1e6;
-                  }
-                `}
-              >
-                <div
-                  css={`
-                    display: flex;
-                    flex-direction: column;
-                    width: 100%;
-                  `}
-                >
-                  <div
-                    css={`
-                      ${theme.row};
-                      justify-content: space-between;
-                      width: 100%;
-                    `}
-                  >
-                    <Link
-                      to={q.link}
-                      css={`
-                        font-size: 0.875em;
-                        color: #a42c90;
-                        font-weight: bold;
-                      `}
-                    >
+              <Query key={q.id} inactive={deletingIds.includes(q.id)}>
+                <Flex flexDirection="column" width="100%">
```

Comment:
isn't `<Flex flexDirection="column"/>` == `<Column/>`?

Reply:
lol yeah, and with the new helpers could also just be `<Flex column width="100%">`... will change to the component though :)